app/helpers/communicart_mailer_helper.rb

```diff
@@ -1,6 +1,12 @@
 module CommunicartMailerHelper
-  def status_icon_tag(status, linear=false)
-    image_tag("icon-#{status}.png", class: "status-icon #{status} #{'linear' if linear}")
+  def status_icon_tag(status, linear_display=false)
+    base_url = root_url.gsub(/\?.*$/,'').chomp("/")
+    bg_linear_image = base_url + image_path("bg_#{status}_status.gif")
+
+    image_tag(base_url + image_path("icon-#{status}.png"),
+              class: "status-icon #{status} #{'linear' if linear_display}",
+              style: ("background-image: url('#{bg_linear_image}');" if linear_display)
```

Comment:
We need to be able to have a variable vertical width to work with any font size. If we hard code the image, we aren't able to make the height variable.